incubator/binding-risingwave.spec/src/main/scripts/io/aklivity/zilla/specs/binding/risingwave/config/proxy.function.yaml

```
@@ -0,0 +1,30 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+---
+name: test
+bindings:
+  app0:
+    type: risingwave
+    kind: proxy
+    options:
+      udf:
+        url: http://localhost:8815
```

We can predict that we'll need separate UDF servers for different languages, such as `java` and `python` over time, even though we are starting with `java` first.

Please make sure we design a syntax that can remain backwards compatible and feels natural when adding `python` UDF server later.

```suggestion
    options:
      udf:
        - server: http://localhost:8815
          language: java # default
```

Initially this array can have max length 1 similar to catalogs.

Later, when we add support for `python`, we can host different user-defined functions on different servers as needed.

```suggestion
    options:
      udf:
        - server: http://udf.zillabase.dev:8815
          language: java # default
        - server: http://udf-python.zillabase.dev:8815
          language: python
```

If needed (even later!), we can further enhance this to host different UDFs on different servers, even for the same language, by adding a pattern for matching UDF names on a specific server.  Then the injection of the `USING LINK` value would be dependent on the UDF name and match a bit like routing.